packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -420,4 +420,39 @@ void main() {
     expect(controller.offset, lessThan(0.0));
     expect(refreshCalled, isTrue);
   }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS,  TargetPlatform.macOS }));
-}
+
+  testWidgets('RefreshIndicator does not force child to relayout', (WidgetTester tester) async {
+    int layoutCount = 0;
+
+    Widget layoutCallback(BuildContext context, BoxConstraints constraints) {
+      layoutCount++;
```

Comment:
nit: our style guide prefers `+=` over `++`: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer--over-